components/support/nimbus-fml/src/command_line/mod.rs

```diff
@@ -101,19 +95,17 @@ fn create_single_file_from_cli(
 }
 
 fn create_generate_command_experimenter_from_cli(
-    matches: &ArgMatches,
+    cmd: &cli::GenerateExperimenter,
     cwd: &Path,
 ) -> Result<GenerateExperimenterManifestCmd> {
-    let manifest = input_file(matches)?;
+    let manifest = cmd.input.clone();
     let load_from_ir =
         TargetLanguage::ExperimenterJSON == TargetLanguage::from_extension(&manifest)?;
-    let output =
-        file_path("output", matches, cwd).or_else(|_| file_path("OUTPUT", matches, cwd))?;
+    let output = cwd.join(&cmd.output);
```

Comment:
I don't believe this changes anything.  `OUTPUT` is a positional argument so there's no reason to check 2 cases.

Reply:
I'm prety sure this is fine.